core/src/identity.rs

```diff
@@ -323,7 +323,7 @@ impl TryFrom<keys_proto::PublicKey> for PublicKey {
             #[cfg(any(not(feature = "rsa"), target_arch = "wasm32"))]
             keys_proto::KeyType::Rsa => {
                 log::debug!("support for RSA was disabled at compile-time");
-                Err(DecodingError::new("Unsupported"))
+                Err(DecodingError::new("RSA keys are unsupported"))
```

Comment:
``` diff
-                Err(DecodingError::new("RSA keys are unsupported"))
+                Err(DecodingError::new("RSA keys are unsupported. Did you enable the `rsa` feature?"))
````

How about mentioning the feature flag here and below?

Reply:
Or maybe "Consider enabling the `rsa` feature.".

Reply:
Thanks, that gave me an idea for a better wording!

Reply:
See [`c81f730` (#2972)](https://github.com/libp2p/rust-libp2p/pull/2972/commits/c81f730aaceef796cae4ebe09155cb318f4095bb).